### ISSUE TITLE
Include outreach events

### ISSUE THREAD

**Issue by DForbush** [Chicago/census2020_ward_rpt#5]
We need to figure out a way to automate the inclusion of outreach events that occur within each ward. This will likely be taken from some kind of google form, where the locations of those events need to be geocoded, then matched with the ward they take place in, and then output as a list in the report. 

Should we also try to limit that list based on date, i.e. include only events that have occurred in the last week or events that are coming up within the next week? Or should it be a comprehensive list?

**Comment by sherryshenker** [comment]
We can help you set part of this up in Platform. You could create the Google Sheet which the wards have access to, set up a daily or weekly import of the Google Sheet to Platform, where the data could be geocoded and matched to wards. I think honestly the most amount of work is around getting the event planners to input the event information. I shared an example of the google form here https://docs.google.com/forms/d/19RK-fE1UZbpS9ctN88U5_DT27I7kjG_CFAvZHyPerx0/edit

**Comment by DForbush** [comment]
Thanks for sharing the draft form Sherry. I'm not an expert in this kind of thing, but I've made some edits to it this morning. Some thoughts: 

- I think the only required information should be date, address, location, and estimated number of attendees (and I'm even on the fence about estimated number of attendees - what if they have no idea?). 
- I think all the demographic/information collection should be optional. If people don't have the time to fill that sort of thing out, it's ok (in my view, at least). So I've re-ordered the questions to make that a little more clear. 
- Also, is the longitude/latitude calculated automatically by the form, or is that going to be something that people have to input manually? I don't think we should ask people to input that and it should be done automatically. But I'm just not sure how that works so I thought I'd ask.

**Comment by geneorama** [comment]
The amount of information requested in the form seems reasonable to me, especially because the demographic information is optional. I'm checking here to see if we could / should use Salesforce to manage the events.

**Comment by sherryshenker** [comment]
@DForbush , I think your comments make sense. You could definitely make more fields optional, as long as we collect an address and name. As for latitude and longitude, I think we can remove those entirely and geocode on our end. 

At this point, I think first priority is finalizing this to get it out to the people who will be filling it out, since events are already happening.

**Comment by DForbush** [comment]
@sherryshenker, Is the Google Form live? Can we start sending it out to nonprofits and alderman and whatnot? What is the correct link to use?

**Comment by DForbush** [comment]
Just talked to @sherryshenker. Yes, the link above is the link to share. It's now final and we can send it out for people to use.

**Comment by geneorama** [comment]
Will events added through that form appear in the CIC? I would like to add the link to the README for this project and close this issue

**Comment by geneorama** [comment]
@sherryshenker or @srao-civis

**Comment by sherryshenker** [comment]
Yes, we can set that up once there is data coming in. Not sure how many events will be happening given the current situation...
